Modules/IO/BioRad/src/itkBioRadImageIO.cxx

```diff
@@ -69,15 +69,13 @@ struct bioradheader
   unsigned char reserved[6];   // 70  6    NOT USED (old ver.=real lens mag.)
 };
 
-typedef enum
-{
+using biorad_notestatus = enum {
   NOTE_STATUS_ALL = 0x0100,
   NOTE_STATUS_DISPLAY = 0x0200,
   NOTE_STATUS_POSITION = 0x0400
-} biorad_notestatus;
+};
 
-typedef enum
-{
+using biorad_notetype = enum {
```

Comment:
And here: `enum biorad_notetype` please!

Reply:
Done